src/components/DatePickerInline/utils.ts

```diff
@@ -58,6 +60,13 @@ export const datePickerMonthStyles: ComponentStylePropWithVariants<
         justifyContent: 'center',
         overflow: 'hidden',
     },
+
+    backgroundColor: 'colors.surface',
+    monthButton: {
+        borderRadius: 46 / 2,
```

Comment:
can you make it a use a token instead?
I'm sure Material Design didn't define this prop to be 46/2.

Reply:
done @shashank1010